examples/multiple_viewer_widget.py

```diff
@@ -0,0 +1,449 @@
+"""
+This is an example on how to have more than one viewer in the same napari window.
+Additional viewers state will be synchronized with the main viewer.
+Switching to 3D display will only impact the main viewer.
+
+This example also contain option to enable cross that will be moved to the
+current dims point (`viewer.dims.point`).
+"""
+
+from copy import deepcopy
+
+import numpy as np
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QDoubleSpinBox,
+    QPushButton,
+    QSplitter,
+    QTabWidget,
+    QVBoxLayout,
+    QWidget,
+)
+from superqt.utils import qthrottled
+from packaging.version import parse as parse_version
+
+import napari
+from napari.components.layerlist import Extent
+from napari.components.viewer_model import ViewerModel
+from napari.layers import Image, Labels, Layer, Vectors
+from napari.qt import QtViewer
+from napari.utils.action_manager import action_manager
+from napari.utils.events.event import WarningEmitter
+from napari.utils.notifications import show_info
+
+NAPARI_GE_4_16 = parse_version(napari.__version__) > parse_version("0.4.16")
```

Comment:
Honestly, I think that examples should only cater to whatever is on "main" at the time the example is added, and certainly don't need to go back more than one stable release. So I would remove this whole check and support only 0.4.16+. It's even fine to only support 0.4.17+.

Reply:
I think that such example should work on last released branch. For other examples we could told people to check version from release time. 

Without this the example will be useless until next release.

Reply:
Well, I'm ok with things only working on main (ie the example will work at the time that it is released), since the example is only on main, but either way, I got confused by the variable name, it should be `NAPARI_GT_4_16` ("Greater Than") rather than `NAPARI_GE_4_16` ("Greater than or Equal"). 

I admit that since we don't have "stable" docs yet it's better if things work on stable. In the near future, the example won't render until after the version is released, so we really should only worry about main.